apps/backoffice-v2/src/pages/Entity/components/EditableDetails/EditableDetails.tsx

```diff
@@ -118,69 +118,112 @@ export const EditableDetails: FunctionComponent<IEditableDetails> = ({
-                        defaultValue={field.value}
-                      >
-                        <FormControl>
-                          <SelectTrigger className="w-full">
-                            <SelectValue />
-                          </SelectTrigger>
-                        </FormControl>
-                        <SelectContent>
-                          {dropdownOptions?.map(({ label, value }) => {
-                            return (
-                              <SelectItem key={value} value={value}>
-                                {label}
-                              </SelectItem>
-                            );
-                          })}
-                        </SelectContent>
-                      </Select>
-                    ) : (
-                      <FormControl>
-                        <Input
-                          type={!format ? (type === 'string' ? 'text' : type) : format}
-                          disabled={!isEditable}
-                          className={ctw(
-                            `p-1 disabled:cursor-auto disabled:border-none disabled:bg-background disabled:opacity-100`,
-                            {
-                              'font-bold text-success': isDecisionPositive(
-                                isDecisionComponent,
-                                field.value,
-                              ),
-                              'font-bold text-destructive': isDecisionNegative(
-                                isDecisionComponent,
-                                field.value,
-                              ),
-                            },
-                          )}
-                          pattern={pattern}
-                          autoComplete={'off'}
-                          {...field}
-                        />
-                      </FormControl>
-                    )}
-                    <FormMessage />
-                  </FormItem>
-                )}
-              />
-            ),
+          {formData?.map(
+            ({ title, isEditable, type, format, pattern, value, valueAlias, dropdownOptions }) => {
+              return (
+                <FormField
+                  key={keyFactory(valueId, title, `form-field`)}
+                  control={form.control}
+                  name={title}
+                  render={({ field }) => {
+                    if (isDecisionComponent && !value) return null;
+
+                    const isInput = [
+                      !isValidUrl(value) || isEditable,
+                      !isObject(value),
+                      !isArray(value),
+                    ].every(Boolean);
+                    const isSelect = isInput && !!dropdownOptions;
+
+                    return (
+                      <FormItem>
+                        <FormLabel>{toStartCase(camelCaseToSpace(title))}</FormLabel>
+                        {(isObject(value) || isArray(value)) && (
+                          <div
```

Comment:
The previous placement of the `JsonDialog` made it so no label was rendering for the field.